src/core/database/query/lists.ts

```diff
@@ -23,3 +33,34 @@ export function calculateTotalAndPaginateList(
 
   return query.asResult<{ items: string[]; total: number }>();
 }
+
+export const defaultSorter: Sorter = (
+  q,
+  sortInput,
+  order,
+  securedProperties,
+  sortValInput
+) => {
+  //The properties that are stored as strings
+  const stringProperties = ['name'];
```

Comment:
This feels like magic. That's why I wanted to pass in the config to this abstraction so that the concrete caller defines the sorting rules based on the properties it knows about for its domain model.

An example: This doesn't work for users which have `firstName` and `lastName`.

Reply:
Not really following your first statement...but what we've been doing in the intermediary step is make the resource specific sortmap: https://github.com/SeedCompany/cord-api-v3/blob/090a4d38e6393daaf41433485b57d63baae117f4/src/components/organization/organization.service.ts#L268-L271

I tried this and user and it does work for `displayFirstName` and `displayLastName`.  `firstName` is a calculated value, which is another complexity.

Reply:
I created this `const stringProperties = ['name']` just to capture the property across resources  that we know are strings so we don't have to make that sortMap for each resource

Reply:
The thing I don't like about this is it's coupling the sorting strategy with the specific database query. Which is why I created in my branch the separate enum. So `name` is sorted case insensitive, and separately it's a secured property or a special relation or something else. 

> `firstName` is a calculated value, which is another complexity.

You're right. We need to maybe get with @sethmcknight to determine what to do for user sorting. I think we have some work to do with display* properties anyways.